gunicorn/workers/gtornado.py

```diff
@@ -66,24 +66,25 @@ def heartbeat(self):
                         pass
                 self.server_alive = False
             else:
-                if not self.ioloop._callbacks:
+                if IOLOOP_PARAMETER_REMOVED:
```

Comment:
Could you also update the comment in line 23?

Reply:
And perhaps we could rename `IOLOOP_PARAMETER_REMOVED` to something more general.

Reply:
Perhaps `TORNADO5_IOLOOP`?

Reply:
That sounds good. We could make it even more generalized with something like `HAS_TORNADO5` or just `TORNADO5`.